sale_force_invoiced/model/sale_order.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = 'sale.order'
+
+    force_invoiced = fields.Boolean(
+        string='Force invoiced',
+        help='When you set this field, the sales order will be considered as '
+             'fully invoiced, even when there may be ordered or delivered '
+             'quantities pending to invoice.',
+        readonly=True,
+        states={'done': [('readonly', False)]},
```

Comment:
I'm wondering if it shouldn't be editable on _sale_ state rather than on _done_ state (or maybe on both).

Reply:
Please @mreficent could you answer this doubt? thanks